daml-lf/governance.rst

```diff
@@ -192,6 +192,10 @@ always considered to be newer than every stable minor version of that
 major version, it must be backward-compatible with all such stable
 versions.
 
+Newly-implemented minor version level features do not themselves entail
+a minor version bump; instead, they must be staged in the *dev* version
+for later release as a stable group of backward-compatible features.
+
```

Comment:
It is hard to read the above... suggesting:
All newly-implemented minor version features have to be staged in the *dev* version for later release as a stable group of backward-compatible features.

Reply:
652191b9a2